internal/node/controller/controller_test.go

```diff
@@ -27,7 +27,7 @@ func setupNodeDBTest(ctrl *gomock.Controller, t *testing.T) (NodeController, *mo
 	mockedManager.EXPECT().CreateDatabase(gomock.Any(), gomock.Any(), gomock.Any()).DoAndReturn(
 		// signature of anonymous function must have the same number of input and output arguments as the mocked method.
 		func(nodeName, schemaName string, initTransitions []hdb.Transition) (hdb.Client, error) {
-			require.Equal(t, 2, len(initTransitions))
+			require.Equal(t, 4, len(initTransitions))
```

Comment:
should we make this a var equal to something instead of a magic number?

Reply:
I don't think it's a big deal to have magic numbers in tests. If it breaks, the person fixing it will have to stop and think what changed before updating it anyways.